### ISSUE TITLE
Rebuild Search Index

### ISSUE THREAD

**Issue by iceboy233** [ponzu-cms/ponzu#337]
This is useful in many ways. Currently there is no way to enable search on an existing type and get search index of the existing items. It also helps eliminating accumulative errors and restoring from backup (#166). A naive implementation can be iterating through existing items and update the search index.

**Comment by nilslice** [comment]
Yes, I agree, it would be quite useful. Are you interested in taking on this feature?
